smif/schema/pre_specified_schema.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
So, this could be minimally validated as a list of objects if we were to add `"items": { "type": "object" }`

Or, using `$ref` to specify that this should be a list of build instructions - 
```json
   "items": {
        "$ref": "build_instruction"
    }
```

With a `definitions` section to define what a `build_instruction` is:
```json
    "definitions": {
        "build_instruction": {
            "type": "object",
            "properties": {
                "asset": {
                    "type": "string"
                },
                "timeperiod": {
                    "type": "integer"
                },
                "new_capacity": {
                    "$ref": "measurable_value"
                },
                "location": {
                    "$ref": "http://json-schema.org/geo"
                }
            },
            "required": ["asset", "timeperiod"]
        },
        "measurable_value": {
            "type": "object",
            "properties": {
                "unit": {
                    "type": "string"
                },
                "value": {
                    "type": "number"
                }
            },
            "required": ["unit", "value"]
        }
    }
```
This also defines a `measureable_value` and refers to the http://json-schema.org definition of a point.